chia/wallet/wallet_node.py

```diff
@@ -638,7 +638,11 @@ def set_server(self, server: ChiaServer) -> None:
     def initialize_wallet_peers(self) -> None:
         self.server.on_connect = self.on_connect
         network_name = self.config["selected_network"]
-
+        try:
+            default_port = self.config["network_overrides"]["config"][network_name]["default_full_node_port"]
+        except Exception:
```

Comment:
`KeyError`?

Reply:
Probably, but I copied it from the full node - https://github.com/Chia-Network/chia-blockchain/blob/f8849e81824e88fd121996df8ac9a3ae89a39b2f/chia/full_node/full_node.py#L452C9-L452C9

Reply:
I changed it - don't really like blanket catchs